content/en/docs/tutorials/directory-structure.md

```diff
@@ -0,0 +1,325 @@
+---
+title: "Notation directory structure for system configration"
+description: "How to configure the directory structure"
+type: docs
+weight: 4
+---
+
+Notation CLI requires local file systems support for the following components. They should be stored in different places per different operating system.
+
+- Notation binary
+- Plugins
+- Configurations
+- Trust stores
+- Trust policies
+- Signing key store
+
+This documentation specifies the recommended directory structure for those components.
+
+## Category
+
+The directories for various components are classified into the following catagories.
+
+| Alias              | Description                                                                         |
+| ------------------ | ----------------------------------------------------------------------------------- |
+| `NOTATION_BIN`     | Directory for executable binaries                                                   |
+| `NOTATION_LIBEXEC` | Directory for binaries not meant to be executed directly by users' shell or scripts |
+| `NOTATION_CONFIG`  | Directory for configurations                                                        |
+
+Although it is recommended to install `notation` with its plugins and default configurations at the system level, it is possible to install at the user level.
```

Comment:
We do not support system level for Beta.1 and RC.1. Can we remove everything related to system level.

Reply:
Good catch. I remove all system-level descriptions from this guide.